src/porepy/numerics/interface_laws/hyperbolic_interface_laws.py

```diff
@@ -71,7 +73,7 @@ def assemble_matrix_rhs(
         # mapping from upper dim cellls to faces
         # The mortars always points from upper to lower, so we don't flip any
         # signs
-        face_to_cell_h = np.abs(pp.fvutils.scalar_divergence(g_master))
+        face_to_cell_h = np.abs(pp.fvutils.scalar_divergence(g_primary))
```

Comment:
I spot an "h"... face_to_cell_primary?

Reply:
Same here, it might be better to introduce 'low' and 'high'-related inconsistencies in a different PR.

Reply:
TODO: See #478